set.mm

```diff
@@ -19990,6 +19990,20 @@ Corresponds to the dual of Axiom (B) of modal logic.  (Contributed by NM,
       KACLZEKAUALAUBLUCUDDEABEGHMACDFIMDENZABCAUEBCLJOPQABDFRACEGRST $.
   $}
 
+  ${
+    $d x y $.
+    cbv2w.1 $e |- F/ x ph $.
+    cbv2w.2 $e |- F/ y ph $.
+    cbv2w.3 $e |- ( ph -> F/ y ps ) $.
+    cbv2w.4 $e |- ( ph -> F/ x ch ) $.
+    cbv2w.5 $e |- ( ph -> ( x = y -> ( ps <-> ch ) ) ) $.
+    $( Version of ~ cbv2 with a disjoint variable condition, which does not
+       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    cbv2w $p |- ( ph -> ( A. x ps <-> A. y ch ) ) $=
+      ( wal weq wb wi biimp syl6 cbv1v equcomi biimpr syl56 impbid ) ABDKCEKABC
+      DEFGHIADELZBCMZBCNJBCOPQACBEDGFIHEDLUBAUCCBNEDRJBCSTQUA $.
+  $}
+
```

Comment:
Should you mention in change-set.txt that this theorem was moved to main?

Reply:
He added cbv2w just 3 days ago to his mathbox, in order to prepare a bigger restructure in main.  In my eyes this can be seen as a single move, and the short-lived presence in his mathbox is just a staging step to see how well  his suggestion is received in the community..